### DISCUSSION TITLE
how to get the item position

### DISCUSSION THREAD

**Discussion by cry-1337** [SmartlyDressedGames/Unturned-3.x-Community#4240] [Q&A] [answered]
How to get the item position, from the client's side. When I get the point of an item through the TakeItem method, it turns out not to be correct if the item has rolled off
RocketMod

**Comment by DanielWillett** [comment, marked as answer]
Not possible as far as I know unless you're able to spawn the item serverside and simulate it but it would cause lag if you do it for every item. (Haven't looked into doing this at all but it could be possible).
